docs/stacks-guide.md

```diff
@@ -0,0 +1,301 @@
+  cluster. We will put Crossplane in this cluster, and we'll refer to it
+  as the control cluster.
+
+## Install the Crossplane CLI
+
+To interact with stacks, we're going to use the [Crossplane
+CLI][crossplane-cli], because it's more convenient. To install it, we
+can use the one-line curl bash:
+
+```
+RELEASE=0.0.1
+curl -sL https://raw.githubusercontent.com/crossplaneio/crossplane-cli/"${RELEASE}"/bootstrap.sh | bash
+```
+
+To use the latest release, you can use `master` as the `RELEASE` instead
+of using a specific version.
+
+## Install and configure Crossplane
+
+To use Crossplane, we'll need to install and configure it. In this case,
+we want to use Crossplane with a cloud provider, so we'll need to
+configure the provider.
+
+### Install Crossplane
+
+The recommended way of installing Crossplane is by using
+[helm][helm-install]. We can grab the most stable version currently
+available by using:
+
+```
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+For more options for installing, including how to install a more
+bleeding-edge version, or how to uninstall, see the [full install
+documentation][crossplane-install-docs].
+
+### Configure support for your cloud provider
+
+Next we'll set up support for our cloud provider of choice! See the
+provider-specific guides:
+
+* [AWS][aws-setup]
+* [GCP][gcp-setup]
+* [Azure][azure-setup]
+
+Then come back here! Don't worry; we'll still be here when you're ready.
+
+Don't see your favorite cloud provider? [Help us add
+support][provider-stack-developer-guide] for it!
+
+## Install support for our application into Crossplane
+
+Now that we've got Crossplane set up and configured to use a cloud
+provider, we're ready to add support for creating WordPresses! We'll do
+this using a Crossplane Stack. For more information about stacks, see
+the [full Stack documentation][stack-docs].
+
+We can use the [Crossplane CLI][crossplane-cli] to install our stack which adds support for
+Wordpress:
+
+```
+kubectl crossplane stack install 'crossplane/sample-stack-wordpress:latest' 'sample-stack-wordpress'
+```
+
+This pulls the stack package from a registry to install it into
+Crossplane. For more details about how to use the CLI, see the
+[documentation for the CLI][crossplane-cli-docs]. For more details about how stacks work behind
+the scenes, see the documentation about the [stack
+manager][stack-manager-docs] and the [stack
+format][stack-format-docs].
+
+## Create a Wordpress
+
+Now that Crossplane supports Wordpress creation, we can ask Crossplane
+to spin up a Wordpress for us. We can do this by creating a Kubernetes
+resource that our Wordpress stack will recognize:
+
+```
+cat > my-wordpress.yaml <<EOF
+apiVersion: wordpress.samples.stacks.crossplane.io/v1alpha1
+kind: WordpressInstance
+metadata:
+  name: my-wordpressinstance
```

Comment:
The `WordpressInstance` will also need to be in the correct namespace. I used this:

```
cat > my-wordpress.yaml <<EOF
apiVersion: wordpress.samples.stacks.crossplane.io/v1alpha1
kind: WordpressInstance
metadata:
  name: my-wordpressinstance
  namespace: app-project1-dev
EOF

kubectl apply -f my-wordpress.yaml
```

Reply:
Should be fixed; I ended up using a slightly inconsistent way of specifying it, but we can always fix that in a future changeset.